packages/framework/src/Component/Elasticsearch/Exception/ElasticsearchAliasUsedByDifferentIndex.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrameworkBundle\Component\Elasticsearch\Exception;
+
+class ElasticsearchAliasUsedByDifferentIndex extends ElasticsearchIndexException
+{
+    public function __construct(string $alias)
+    {
+        parent::__construct(sprintf(
+            'There is an index for alias "%s" already. You have to migrate it first due to different definition.',
```

Comment:
Index for "%s" already exists. Run `php phing elasticsearch elasticsearch-index-migrate` to update your Elasticsearch index.

Reply:
Disagree. You can run it via symfony console command even with an optional argument for specific index. When doing so, your message would recomend you bad command.

Reply:
How would you descibe that? I want to avoid to print all posibilites how to migrate structure.

Reply:
Index for alias "%s" already exists.